design/src/components/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Footer = () => {
+
+ return (
+  <footer className="footer">
+   <div className="footer-item first">
+    <p>GLÖGGA AB</p>
+    <p>Glöggvägen 24<br />123 45 Glöggköping</p>
+    <p>010-11 22 333<br />kundservice@glogga.se</p>
+   </div>
+   <div className="footer-item second">
+    <a href="#">OM OSS</a>
+    <a href="#">JOBBA HÄR</a>
+    <a href="#">KÖPVILLKOR</a>
+    <a href="#">FAQ</a>
+   </div>
+   <div className="footer-item third">
+    <a href="#"><img src="images/instagram.png" alt="" /></a>
```

Comment:
These icons should benefit from an alt text (in Swedish). [Here](https://accessibility.huit.harvard.edu/describe-content-images) are some guidelines to start with. Also, consider adding the width and height attributes, [see info here](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/).